WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -192,20 +193,11 @@ private async Task DoDequeueAsync(IEnumerable<CoinViewModel> selectedCoins)
 
 				try
 				{
-					await Global.ChaumianClient.DequeueCoinsFromMixAsync(selectedCoins.Select(c => c.Model).ToArray(), "Dequeued by the user.");
+					await Global.ChaumianClient.DequeueCoinsFromMixAsync(selectedCoins.Select(c => c.Model).ToArray(), DequeueReason.UserRequested);
 				}
 				catch (Exception ex)
 				{
 					Logger.LogWarning(ex);
-					var builder = new StringBuilder(ex.ToTypeMessageString());
-					if (ex is AggregateException aggex)
-					{
-						foreach (var iex in aggex.InnerExceptions)
-						{
-							builder.Append(Environment.NewLine + iex.ToTypeMessageString());
-						}
-					}
-					NotificationHelpers.Error(builder.ToString());
```

Comment:
This is the fix for https://github.com/zkSNACKs/WalletWasabi/issues/2678. Now the dequeue subscription in the global will cover this notification instead.